CogsSDK/Classes/PubSubService.swift

```diff
@@ -16,19 +16,16 @@
  */
 
 import Foundation
-import CryptoSwift
 
 /// Opens a connection to Cogswell pub/pub system.
 public final class PubSubService {
-    
+
     /// Creates and configures a pub/sub connection.
     ///
-    /// - Parameters:
-    ///   - keys: The provided project keys.
-    ///   - options: The connection options.
+    /// - Parameter socket: socket instance
     /// - Returns: Returns a configured pub/sub connection handler to manage the connection.
-    public static func connect(keys: [String], options: PubSubOptions?) -> PubSubConnectionHandle {
-        return PubSubConnectionHandle(keys: keys, options: options)
+    public static func connect(socket: Socket) -> PubSubConnectionHandle {
```

Comment:
This is not acceptable. It does not follow the signature given for the PubSubService.connect method. You need to revert this line back to the previous version. It is OK to change the constructor to PubSubConnectionHandle because it is not specified in the signatures document. It is OK to not have any unit tests for PubSubService if you cannot do them without passing in a socket to the constructor. It is not OK to change the signature.